cargo-screeps/src/orientation.rs

```diff
@@ -3,21 +3,16 @@ use std::{env, path::PathBuf};
 use failure;
 
 pub fn find_project_root() -> Result<PathBuf, failure::Error> {
-    // TODO: is it right to canonicalize here?
-    let mut current = env::current_dir()?.canonicalize()?;
+    let orig = env::current_dir()?;
+    let mut current = orig.clone();
 
     loop {
-        if current.join("screeps.toml").exists() {
+        if current.join("Cargo.toml").exists() {
```

Comment:
One reason I was checking for screeps.toml here originally is in order to support correctly building if we're running in a sub-directory which is also a crate (most likely a library used by the top-level deployed crate).